auth0/src/test/java/com/auth0/android/provider/WebAuthProviderTest.java

```diff
@@ -110,13 +116,19 @@ public void setUp() throws Exception {
         prepareBrowserApp(true, null);
     }
 
+    //** ** ** ** ** **  **//
+    //** ** ** ** ** **  **//
+    //** LOG IN  FEATURE **//
+    //** ** ** ** ** **  **//
+    //** ** ** ** ** **  **//
+
     @SuppressWarnings("deprecation")
     @Test
     public void shouldInitWithAccount() throws Exception {
```

Comment:
will probably rename all these old test methods with a prefix for "auth" or "login" e.g. `shouldInitLoginWithAccount`

Reply:
Going to do this on a different PR